### PR TITLE
fix 'cannot scream while unconscious' bug

### PR DESCRIPTION
**Pull request by checkraisefold**
:cl: zennerx
fix: fixed a bug that made you try and scream while unconscious due to a fire
/:cl:

so player's don't get spam saying you can't cough and scream while unconscious when on fire

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -1472,7 +1472,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 					burn_damage = HEAT_DAMAGE_LEVEL_2
 		burn_damage *= heatmod
 		if((prob(burn_damage) * 10) / 4)	//40% for level 3 damage on humans
-			H.emote("scream")
+			if (h.stat != UNCONSCIOUS)
```
Comment (XDTM) [inline code/modules/mob/living/carbon/human/species.dm]: H instead of h

Comment (checkraisefold) [reply code/modules/mob/living/carbon/human/species.dm]: whoops.

**Comment by optimumtact** [comment]
For your changelog, the "optional name here" field can either be removed, in which case it will use your github username, or replaced with your chosen string.

you should not use "optional name here"

**Comment by checkraisefold** [comment]
fixed @optimumtact

**Comment by optimumtact** [comment]
Thanks and welcome to the repo

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -1472,7 +1472,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 					burn_damage = HEAT_DAMAGE_LEVEL_2
 		burn_damage *= heatmod
 		if((prob(burn_damage) * 10) / 4)	//40% for level 3 damage on humans
-			H.emote("scream")
+			if (H.stat != UNCONSCIOUS)
```
Comment (ShizCalev) [inline code/modules/mob/living/carbon/human/species.dm]: Merge this into the above if

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species.dm (hunk 2 above)
Comment (checkraisefold) [reply code/modules/mob/living/carbon/human/species.dm]: thought I did but missed it, fixing.

Comment (checkraisefold) [reply code/modules/mob/living/carbon/human/species.dm]: it's already merged into the above if, it's indented

Comment (vuonojenmustaturska) [reply code/modules/mob/living/carbon/human/species.dm]: he means change the if above to read `if(H.stat != UNCONSCIOUS && (prob(burn_damage) * 10) / 4))`, also you should change the condition to be either `!H.stat` or `H.stat > UNCONSCIOUS`, the former is a fairly generic "is not dead, unconsciout or in crit" check, the latter allows for soft crit as well as conscious.

Comment (checkraisefold) [reply code/modules/mob/living/carbon/human/species.dm]: thanks! fixing it now. i'm pretty new to BYOND coding and especially to checking human properties. not familiar with stat, but familiar with if operators.

**Pull request merged**